app/features-json/project_json_controller.rb

```diff
@@ -25,5 +26,16 @@ class ProjectJSONController < AuthenticatedControllerBase
 
       return project_view_model.to_json
     end
+
+    get "#{HOME}/repos" do
```

Comment:
Repos are attached to the configured robot accounts right? Maybe we should put this in a Repo Controller since it's not specific to a project.

Reply:
Yeah! That makes sense! I'll tackle all the review comments today!